code/game/turfs/simulated/chasm.dm

```diff
@@ -9,6 +9,7 @@
 	canSmoothWith = list(/turf/open/floor/fakepit, /turf/open/chasm)
 	density = TRUE //This will prevent hostile mobs from pathing into chasms, while the canpass override will still let it function like an open turf
 	bullet_bounce_sound = null //abandon all hope ye who enter
+	CanAtmosPass = ATMOS_PASS_NO
```

Comment:
But... Isn't it just a... hole? Why wouldn't atmos pass?

Reply:
pretty sure the whole reason this happened in the first place is because chasms would repeatedly suck in air that was being generated from lavaland tiles

Reply:
they shouldn't?

Reply:
consider this a revert of part of #8977 then